doc/nrf/ug_thread_overview.rst

```diff
@@ -44,3 +44,12 @@ This includes:
 
 * All device roles
 * Border Router support
+
+To stay up to date with the newest Thread trends, the |NCS| also gradually implements features introduced in Thread 1.2 version.
+This protocol version is fully backward compatible with Thread 1.1 (including Thread 1.1.1) and improves networks E.g. by making it more scalable, capable, responsive and low-power.
+
+For more information about Thread 1.2 see official materials at `Thread Group's Thread 1.2 Commercial White Paper`_ and `Thread Group's Thread 1.2 Base Features`_.
+
+You may choose the version of Thread protocol, which will be used in your application.
+By default |NCS| supports Thread 1.1, but you can enable and configure Thread 1.2 using dedicated options (see :ref:`thread_ug_thread_1_2` for details).
+|NCS| does not support all Thread 1.2 functionalities yet, so under above link you can also find the list of currently available features with information on how to enable it.
```

Comment:
I would mention about the currently available functionalities in the configuration section.